docs/api-docs/storefront/graphql/graphql-api-overview.mdx

```diff
@@ -3,34 +3,34 @@
 <Callout type="info">
 #### Early access
 The GraphQL Storefront API is in early access and is feature-incomplete. It will remain in early access until we reach the minimum amount of functionality necessary to power an end-to-end shopping experience. As new features are added to the API, we'll announce them in our [Developer Changelog](/changelog#labels/storefront-api).
+
+"Early access" is not a measure of the API's quality, stability, or readiness for use in a production capacity on transactional storefronts. This is a production API which is fully supported & tested by BigCommerce, and you should be comfortable using it if your use case is satisfied by the current functionality.
 </Callout>
 
-BigCommerce's GraphQL Storefront API makes it possible to query storefront data from within a [Stencil](/stencil-docs/getting-started/about-stencil) theme or remote site. This means information previously only available on the back-end using [Stencil's template logic](/stencil-docs/reference-docs/global-objects-and-properties) can now be accessed with front-end JavaScript. For example, with the GraphQL Storefront API, it is possible to do the following:
+BigCommerce's GraphQL Storefront API makes it possible to query storefront data from within a [Stencil](/stencil-docs/getting-started/about-stencil) theme, or from external websites or applications in a headless shopper experience. This GraphQL API serves as a flexible interface for building shopper experiences. For example, with the GraphQL Storefront API, it is possible to do the following:
 
-* Access product options, variations, and custom fields for any product from any page.
```

Comment:
This verbiage was more relevant when this use case was completely unsolved, but now the tribal knowledge has spread (and the API has a lot more functionality) so it's not the most exciting callout anymore.